code/math/floating.h

```diff
@@ -97,10 +97,9 @@ float fl_isqrt( float x )
 */
 
 // sees if a floating point number is within the minimum tolerance of zero
-inline bool fl_near_zero(float a)
+inline bool fl_near_zero(float a, float e = std::numeric_limits<float>::epsilon())
```

Comment:
Since this is adding a new parameter, change the comment to be this:
```
// sees if a floating point number is within a certain threshold (by default, epsilon) of zero
```

Reply:
Good idea, just added.